packages/vaadin-icon/src/vaadin-iconset.d.ts

```diff
@@ -15,11 +15,16 @@ declare class IconsetElement extends ElementMixin(HTMLElement) {
    */
   name: string;
 
+  /**
+   * The size of an individual icon. Note that icons must be square.
+   */
+  size: number;
+
   /**
    * Produce SVGTemplateResult for the element matching `id` in this
```

Comment:
nit: did you mean `name`?
```suggestion
   * Produce SVGTemplateResult for the element matching `name` in this
```

Reply:
Thanks, fixed.